### ISSUE TITLE
Plugin doesn't seem to work for sites using Servd Asset Storage

### ISSUE THREAD

**Issue by chrismlusk** [engram-design/ImageResizer#76]
### Describe the bug

Images are not resized at all on my site, which is hosted on Servd.

### Steps to reproduce

1. Install plugin on a site hosted on Servd.
2. Try uploading a photo or using the batch resizing action.

### Craft CMS version

4.4.10

### Plugin version

3.0.6

### Multi-site?

Yes

### Additional context

I first installed and configured the plugin locally, and it only partly works. 

Locally, images are resized as expected on upload. 

However, using the batch resize feature does not work. When attempting to do so, I get this error in the logs:

```
{
  "dateTime": "2023-05-16 09:32:28",
  "taskId": "CT0FR",
  "handle": "error",
  "filename": "pexels-marcus-aurelius-6787870.jpg",
  "data": {
    "message": "finfo_file(\/var\/www\/html\/app\/storage\/runtime\/assets\/sources\/910431.jpg): Failed to open stream: No such file or directory"
  }
}
```

**Comment by engram-design** [comment]
I'll have to do some testing here, but last time I checked with non-local filesystems this _was_ working (S3 for example), but maybe there's something specific with Servd.

Are you using the Servd asset storage locally as well? As in - the batch resize fails even on local, not just on their staging/production environments?

**Comment by chrismlusk** [comment]
That’s right. I am using Servd asset storage locally as well.

**Comment by chrismlusk** [comment]
Hi Josh. Let me know if I can get you any other info about this. Thanks.

**Comment by engram-design** [comment]
Sorry, I haven't had the time to spin up a Servd site and run some debugging yet, I'll try my best to look into it!

**Comment by steverowling** [comment]
Is there any update on this? We're seeing the same problem, which is frustrating as the client's asset storage is filling up quickly and will trip them over into needing to buy more storage from Servd soon, just because we can't reliably resize their uploaded images. This is for a Craft 3 site.

**Comment by engram-design** [comment]
Should be fixed in [3.0.7](https://github.com/verbb/image-resizer/releases/tag/3.0.7) and [2.2.1](https://github.com/verbb/image-resizer/releases/tag/2.2.1)
